internal/pkg/ccloudv2/flink.go

```diff
@@ -88,5 +88,5 @@ func (c *Client) ListFlinkIAMBindings(environmentId, region, cloud, identityPool
 		req = req.IdentityPool(identityPoolId)
 	}
 	res, httpResp, err := req.Execute()
-	return res.GetData(), errors.CatchCCloudV2Error(err, httpResp)
+	return res.GetData(), errors.CatchOrgV2ResourceNotFoundError(err, environmentId, httpResp)
```

Comment:
The function is called CatchOrgV2... but it's being used in a FlinkV2 function. Can we rename it since we're using it for more than just OrgV2 now?

Reply:
```suggestion
	return res.GetData(), errors.CatchCCloudV2ResourceNotFoundError(err, environmentId, httpResp)
```

Reply:
Of course! thanks for the suggestion @brianstrauch ! Updated!